src/provable/curve-bigint.ts

```diff
@@ -1,5 +1,5 @@
-import { Bool, Field } from './field.js';
-import { provable } from './provable.js';
+import { Bool, Field } from './field-bigint.js';
```

Comment:
this file is integrated into snarkyjs by this PR and should be reviewed in its entirety!